SignalMessaging/attachments/ApproveContactShareViewController.swift

```diff
@@ -277,7 +296,16 @@ public class ApproveContactShareViewController: OWSViewController, EditContactSh
 
     // TODO: Surface error with resolution to user if not.
```

Comment:
I believe this PR contains all of the SAE-specific work for contact sharing. Any remaining work is in classes used in contact sharing in main app and SAE.